tests/test_global_opts.rs

```diff
@@ -156,8 +156,8 @@ color="always""#,
     );
     let stdout = test_env.jj_cmd_success(&repo_path, &["log", "-T", "commit_id"]);
     insta::assert_snapshot!(stdout, @r###"
-    @ [1;34m230dd059e1b059aefc0da06a2e5a7dbf22362f22[0m
```

Comment:
This output changed slightly (the color is unchanged).

![image](https://user-images.githubusercontent.com/4123047/211689430-b0d36da7-db36-4281-9fb5-ff4214fcc4de.png)

Reply:
Yeah, that depends on your terminal. Which one are you using? I think I tried on iTerm2 and xterm and they both rendered the old format as bold and bright even though we only said that it should be bold. Your terminal seems to make "bold" only bold. Since it's so common for terminals to render bold as bold+bright, it felt like a regression to lose the bright colors on those terminals.

By the way, I have noticed another regression in this PR, which is that there's new an unwanted blank line after the for line of the description in the working-copy commit. I'll look into that later, but expect an update to this PR.

Reply:
All the other tests render correctly. I think there's something (very mildly) wrong here. I see it in both Kitty and Gnome Terminal.

It is correct for terminals to render text with the bold attribute as bright, I think. However, some terminal color themes use the same color for bright as for non-bright, making the text just bold.

By the way, my current favorite way of looking at these is to add `.diff` to the github's commit URL, run the `sponge | less -R` command from `moreutils` in the terminal, paste the content there, and press `^D`. If you have these files on disk, you can just run `less -R` on your files.

Reply:
Which commit is that diff from? Whichever it is, the old output is not bright but the new output is bright. My guess is that you're looking at the commit that started using `crossterm`. From that commit, we explicitly set bright color and bold font.

Reply:
Actually, I have no reason to claim this is wrong. Just weird. Whatever was wrong is actually fixed by this commit. Not sure what's going on or whether it's worth investigating.  So, never mind.

Reply:
Didn't see your response before last comment. I think we're in agreement. Yes, I was looking at the `crossterm` commit.

Reply:
There's a discussion about that: https://codeberg.org/dnkl/foot/issues/199. According to the author, "terminals are moving away from drawing bold text as bright", though it's not true for any of the ones I use.